libraries/AP_Math/AP_Math.cpp

```diff
@@ -195,7 +195,7 @@ int wrap_360(const int angle)
 float wrap_360_cd(const float angle)
 {
     float res = fmodf(angle, 36000.0f);
-    if (res < 0) {
+    if (res < 0.0f) {
```

Comment:
tridge might fight you on these changes.

OTOH, maybe not given we're thinking about bringing in double support in the EKF, so telling the size of your constants might be important again.

Reply:
that mostly because my IDE thrigger a warning that we compare float and double. I haven't checked in build with double enable if that cause an issue